Simcoe/Analytics Tracking Protocols/LifetimeValueIncreasing.swift

```diff
@@ -6,6 +6,7 @@
 //  Copyright © 2016 Prolific Interactive. All rights reserved.
 //
 
+@available(*, deprecated, message: "Use LifetimeValueTracking instead", renamed: "LifetimeValueTracking")
```

Comment:
Didn't know whether to simply remove the file or have a compiler warning appear (went with the latter with suggestion to update)